modules/33acrossAnalyticsAdapter.js

```diff
@@ -235,7 +237,25 @@ function enableAnalyticsWrapper(config) {
     auctions: {},
   };
 
+  if (isGptPubadsDefined()) {
+    subscribeToGamSlots();
+  } else {
+    window.googletag = window.googletag || {};
+    window.googletag.cmd = window.googletag.cmd || [];
+    window.googletag.cmd.push(subscribeToGamSlots);
+  }
+
   analyticsAdapter.originEnableAnalytics(config);
+
+  function subscribeToGamSlots() {
```

Comment:
Although I know that disabling and enabling the adapter multiple times is not a common scenario, my recommendation is to move `subscribeToGamSlots() {` out of the `enableAnalyticsWrapper` function in order to prevent the function to be redefined.

Reply:
makes sense, moved